tslint.json

```diff
@@ -15,7 +15,8 @@
               "check-strings": true,
               "check-regex": true
             }
-          ]
+          ],
+          "quotemark":["true", "tick", "backtick"]
```

Comment:
This was added to allow backticks in places where normal ticks should have otherwise been used, e.g. in multiline situations.